### PR TITLE
Remove verbosity argument from learning rate schedulers

### PR DESCRIPTION
**Pull request by anwai98**
This PR removes the `verbose` argument from the learning rate schedulers as it is depreciated in `pytorch>=2.7`, and the latest pytorch is up on `conda-forge` since last week!

GTG from my side!

### CODE REVIEW

File: micro_sam/training/training.py
Comment (anwai98) [inline micro_sam/training/training.py:1]: I think this is the most important spot to take care of, however I updated all scripts in reach with this argument.

**Comment by anwai98** [comment]
I'll go ahead and merge this once the tests pass!

PS. @constantinpape I kept this PR on `master` as users would get the latest `pytorch` now onwards, and the training would break for them, as `pytorch` simply remove the argument from the signature at all.
